.gitlab-ci.yml

```diff
@@ -34,6 +34,15 @@ build-verify:
     - docker push $APP_DEV_BASE_IMAGE
     - docker push $APP_PROD_IMAGE
 
+rspec:
```

Comment:
do you mean app-dev-base? rspec need a test database to run but currently this service doesnt have it, or shall I add a db link to it?